src/service/cia301/co_csdo.h

```diff
@@ -226,6 +362,14 @@ CO_ERR COCSdoRequestDownload(CO_CSDO *csdo,
                              CO_CSDO_CALLBACK_T callback,
                              uint32_t timeout);
 
+CO_ERR COCSdoRequestDownloadBlock(CO_CSDO *csdo,
```

Comment:
Add function comment

Reply:
removed the function and added a new parameter to the COCSdoRequestDownload.